antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/SqlBase.g4

```diff
@@ -110,6 +110,7 @@ statement
       cqSelectIntoClause #createContinuousQueryStatement
     | DROP (CONTINUOUS QUERY | CQ) continuousQueryName=ID #dropContinuousQueryStatement
     | SHOW (CONTINUOUS QUERIES | CQS) #showContinuousQueriesStatement
+    | CREATE STORAGE GROUP prefixPath #createStorageGroup
```

Comment:
Move this next to `SET STORAGE GROUP ...` to put the same type command together.

Reply:
Thanks, new commit add integration test and move sql